tee-worker/litentry/pallets/identity-management/src/tests.rs

```diff
@@ -46,6 +46,13 @@ fn set_user_shielding_key_works() {
 #[test]
 fn create_identity_works() {
 	new_test_ext().execute_with(|| {
+		let shielding_key: UserShieldingKeyType = [0u8; USER_SHIELDING_KEY_LEN];
+		assert_ok!(IMT::set_user_shielding_key(
+			RuntimeOrigin::signed(ALICE),
+			BOB,
+			shielding_key.clone()
+		));
+
```

Comment:
If this needs to be done for every case, we can add it to `new_test_ext` in `mock.rs` to avoid duplicate code

Reply:
👌